Resources/Prototypes/_CM14/Catalog/Fills/Clothing/belt.yml

```diff
@@ -5,11 +5,11 @@
   components:
   - type: StorageFill
     contents: #todo cm14 port tools
-      - id: Crowbar
-      - id: Wrench
-      - id: Screwdriver
-      - id: Wirecutter
-      - id: Welder
+      - id: CMCrowbar
+      - id: CMWrench
+      - id: CMScrewdriver
+      - id: CMWirecutter
+      - id: CMWelder
       - id: Multitool
```

Comment:
Cm13 multitool is named the security access tuner; I assume you didnt add that one yet